src/com/google/javascript/jscomp/CoverageInstrumentationPass.java

```diff
@@ -154,4 +164,9 @@ private Node createConditionalObjectDecl(String name, Node srcref) {
                     jscovData)));
     return var.useSourceInfoIfMissingFromForTree(srcref);
   }
+
+  public enum CoverageReach {
```

Comment:
What is this enum for? I don't see any usages of it in among changed lines.

Reply:
The auto-formatter for whatever reason moved it down here. I'm not sure why but I undid it. I never worked with this enum.